src/metrics/metrics.md

```diff
@@ -1,11 +1,62 @@
 # Metrics
 
-Here we provide definitions and information about our standard metrics.
+Here we provide definitions and information about our standard metrics.  We use some terminology that is defined [here](https://docs.telemetry.mozilla.org/concepts/terminology.html).
 
-Table of contents with links to all metrics.
+## Table of Contents
+
+<!-- toc -->
+
+-----
+
+## Daily Active Users (DAU)
+
+### Overview
+
+Daily Active Users or DAU counts the number of unique profiles active in the product on each day.  This is intended to approximate the number of people using the product each day.
+
+### Details
+
+The concept of a "unique profile" may vary from product to product and the relationship between that concept and actual users may also vary.  As such care should be taken interpreting this metric as an actual count of users.
+
+The concept of being "active" is defined as in the [`usage criterion`](./usage.md) section and will vary by product.  A product may even have multiple possible `usage criteria`, but there will be one default choice that is our best approximation of general product usage.
```

Comment:
Here's a draft revision given the conversation in #msg-internal

```suggestion
By default, active users are users who've sent a `main` ping (on desktop) or a `baseline` ping (on mobile).

This is a pretty low bar to call a user "active", so we'll occasionally restrict the metric to users with more activity. It's **critical to clearly document any additional qualifications on the metric**. The metrics team suggest the following format: `DAU(qualification)`. For example, we might be interested in the count of users who've viewed more than 5 URIs in a day. We'd denote that metric as `DAU(URI > 5)`. 
```

Reply:
In principle I'm fine with this phrasing. IMO it would be better to use the "profiles" terminology here, to reinforce that a unique profile (client) is not the same as a user. Something like:

`By default, "active users" are defined as unique profiles who have sent a `main` ping (on desktop) or a `baseline` ping (on mobile).`